src/MediaRouter.php

```diff
@@ -23,12 +23,17 @@ public function media()
                 'search' => false,
             ];
 
+            $options = array_column($options, 'value');
+
+            $options = array_combine(array_values($options), array_values($options));
```

Comment:
looks strange for me, maybe we need to call array_fill_keys here? 

please also mo this logic into the `else` block